recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "kikuchipy" %}
-{% set version = "0.3.1" %}
+{% set version = "0.3.0" %}
```

Comment:
Is this not supposed to be 0.3.1?

Reply:
Thanks for pointing this out! Turns out that this was really a 0.3.0 release, not a 0.3.1. Will make a new PR.